string2.py

```diff
@@ -43,8 +50,8 @@ def not_bad(s):
 # Given 2 strings, a and b, return a string of the form
 #  a-front + b-front + a-back + b-back
 def front_back(a, b):
-    """Your code goes here.  Edit this docstring."""
-    return
+    #length of (a) divided by two rounds down, making it a negative rounds it "up", and then change the negative into a positive using a "-" gives me the indexes i need
+    return a[:-(-len(a)//2)] + b[:-(-len(b)//2)] + a[-(-len(a)//2):] + b[-(-len(b)//2):]
 
```

Comment:
This is a bit hard to read.  Use of floor division operator // is good, but not so much the forced negation.  Can you test for odd/even and adjust by +1 instead?